internal/blocksync/pool.go

```diff
@@ -536,6 +541,11 @@ func (peer *bpPeer) onTimeout() {
 
 //-------------------------------------
 
+type BlockResponse struct {
```

Comment:
Consider renaming this to `TrustedBlockData` or something similar

Reply:
The name was left over from the rpevious implementation. Renamed it.